pages/api/pokemon.ts

```diff
@@ -0,0 +1,106 @@
+import { NamedResource } from '@/types/common';
+import {
+    Pokemon,
+    PokemonListResponse,
+    PokemonSpecies,
+    PokemonSummary,
+    PokemonType,
+} from '@/types/pokemon';
+import { getMaxChunks, getNextChunk } from '@/utils/pokemon-chunks';
+
+export const BASE_URL = 'https://pokeapi.co/api/v2/pokemon';
+export const IMAGE_BASE_URL =
+    'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork';
+export const WIKI_URL = 'https://bulbapedia.bulbagarden.net/wiki';
+
+export async function getPokemonChunk(offset: number = 0, limit: number = 12) {
+    const response = await fetch(
```

Comment:
This is mostly a personal choice. I like the `fetch` API more than `axios` because I feel it's a little simpler and it comes already built-in so no imports necessary (unless you use it in a backend Node application)